lib/queries/taxon_name/filter.rb

```diff
@@ -562,6 +573,38 @@ def leaves_facet
         leaves ? ::TaxonName.leaves : ::TaxonName.not_leaves
       end
 
+      # @return Scope
+      def latinized_facet
+        return nil if latinized.nil?
+
+        tnc = ::TaxonNameClassification.arel_table
+        if latinized == true
+          # Note the query here does not restrict to genus/species groups - a
+          # genus whose rank is changed to subfamily can retain its gender,
+          # e.g., and we want to include those here.
+          ::TaxonName.where(
+            ::TaxonNameClassification.where(
+              tnc[:taxon_name_id].eq(::TaxonName.arel_table[:id]).and(
```

Comment:
@kleintom When we originally coded out this aproach I leaned heavily on arel, I've since moved somewhat away from that, for better or worse.  This is fine (because of specs). Do check to see if there are some generic methods in superclasses (there used to be `def table` which gives you the `TaxonName.arel_table`, for example).

The `.with()` paradigm makes it significantly easier to join individual queries as well.

There are certainly other considerations I've realized in the Filter framework that we could discuss in person perhaps, things to think about in terms of general optimizations.